php/php.editor/src/org/netbeans/modules/php/editor/typinghooks/PhpCommentGenerator.java

```diff
@@ -541,7 +541,7 @@ public void scan(ASTNode node) {
                                                         case INT:
                                                             type = Type.INTEGER;
                                                             break;
-                                                        case REAL:
```

Comment:
Is it safe to remove this line? Should not we remove it in the future, together once the `REAL` type is removed?

CC @junichi11

Reply:
It should be OK (I assume all `REAL` types were replaced with `FLOAT` in the parser).
But I would leave it, just in case.